Nuro/Utilities/CoreData/CoreDataManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  CoreDataManager.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 11/10/22.
+//
+
+import Foundation
+import CoreData
+
+class CoreDataManager {
+    
+    static let shared = CoreDataManager()
+    let container: NSPersistentContainer
```

Comment:
Assess on `NSPersistentCloudKitContainer`. CloudKit usage should be using CloudKit Container